account_invoice_line_sale_line_position/models/account_invoice.py

```diff
@@ -25,11 +25,10 @@ class AccountMoveLine(models.Model):
     @api.depends("sale_line_ids.position")
     def _compute_position_formatted(self):
         for record in self:
-            if record.display_type:
+            if record.display_type != "product":
```

Comment:
I was unsure why this condition is relevant at all, in theory the code below should be sufficient. In my case, just dropping, it would work as well.

Maybe someone has insights why it was added in first place?

Reply:
It's like checking if you have `sale_line_ids` so in the end it shouldn't change much.
Probably it was fundamental in v14